### PR TITLE
Convert most other scenes to RayPerception sensor

### PR DESCRIPTION
**Pull request by chriselion**

### CODE REVIEW

File: UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs [hunk 1]
```diff
@@ -3,6 +3,13 @@
 
 public class AgentSoccer : Agent
 {
+    // Note that that the detectable tags are different for the blue and purple teams. The order is
```
Comment (chriselion) [inline UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs]: Not sure where else to document this. It's a bit non-intuitive if you just look at the tags in the sensor.

Comment (surfnerd) [reply UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs]: seems pretty self documenting in the editor.  Having it here only makes it more clear.

**Comment by chriselion** [comment]
Need to retrain the models for these - will kick off a cloud run over the weekend.

**Comment by chriselion** [comment]
Having trouble getting Pyramid cloud training to run. Backing that out for now.

**Pull request merged**